Dockerfile

```diff
@@ -1,49 +1,37 @@
-FROM node:16-slim
+#
+# Build
+#
+FROM node:16-slim AS builder
 
-# Install node_modules into a different directory to avoid npm/npm#9863.
-RUN mkdir -p /srv/node
-COPY package.json /srv/node/
-COPY yarn.lock /srv/node/
 WORKDIR /srv/node
+COPY package.json yarn.lock /srv/node/
+
+RUN yarn install --pure-lockfile
+
+#
+# Install
+#
+FROM node:16-slim
+
+ARG app_uid=9500
```

Comment:
Why `9500`?

Reply:
Because `addons-server` uses it and I want to keep the UID consistent across the components.